action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -62,11 +62,13 @@ type (
 		suicideSnapshot     map[int]deleteAccount // snapshots of suicide accounts
 		preimages           preimageMap
 		preimageSnapshot    map[int]preimageMap
+		logsSnapshot        []int // logs is an array, save len(logs) at time of snapshot suffices
```

Comment:
shall we revert transaction logs as well? if so, it is better to have it fixed in this PR as well.

Reply:
right, good catch